python/mlx/optimizers.py

```diff
@@ -105,6 +109,8 @@ def apply_single(
             return parameter - self.learning_rate * gradient
 
         v = state.get("v", mx.zeros_like(gradient))
+        if self.nesterov:
+            gradient += self.momentum * v
         v = self.momentum * v + (1 - self.momentum) * gradient
```

Comment:
Which version of Nesterov are you going for? The original in the paper or the PyTorch flavor? This doesn't look like the [PyTorch version](https://pytorch.org/docs/stable/generated/torch.optim.SGD.html) to me.

Also I'm just noticing this now, but I think our momentum is slightly non-standard in that we scale the gradient by `(1-self.momentum)`. Should we change that? (CC @angeloskath ).

Reply:
Agreed. Going directly with the PyTorch version of both would be the simplest.